File: server/order-service/src/main/java/com/sudo248/orderservice/controller/order/dto/CartDto.java
package com.sudo248.orderservice.controller.order.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class CartDto {
    private String cartId;
    private Double totalPrice;
    private int quantity;
    private String status;
    private List<OrderCartProductDto> cartProducts;
}

File: client/mobile/Sudoo/app/src/main/java/com/sudo248/sudoo/data/dto/cart/CartDto.kt
package com.sudo248.sudoo.data.dto.cart

data class CartDto(
    val userId: String = "",
    val cartId: String = "",
    var totalPrice: Double = 0.0,
    var quantity: Int = 0,
    val status: String = "",
    var cartProducts: List<CartProductDto> = listOf()
)







